shell/browser/api/atom_api_app.h

```diff
@@ -113,6 +114,7 @@ class App : public AtomBrowserClient::Delegate,
 #endif
 
   // content::ContentBrowserClient:
+  base::FilePath GetFontLookupTableCacheDir() override;
```

Comment:
Can you move this to `AtomBrowserClient` ? The `App` delegate class is only used for user intervention.